student.rb

```diff
@@ -1,16 +1,15 @@
-class Student
+require_relative 'SeniorTeacher'
+
+class Student < SeniorTeacher
```

Comment:
A student is a senior teacher, too? That REALLY doesn't sound right to me!  If none of the existing classes make sense as a superclass, create a new class to be a superclass of both!